model-building.Rmd

```diff
@@ -4,8 +4,7 @@
 
 En el capítulo previo tu aprendiste como funcionan los modelos lineales, y aprendiste algunas herramientas basicas para entender lo que un modelo esta diciendo con sus datos. El capitulo previo se enfocó en simular conjunto de datos. Este capitulo se centrará en datos reales, mostrando como tu puedes progresivamente construir un modelo ayudando a entender los datos. 
 
-**Tomaremos ventaja del hecho que se puede (que podemos) pensar que un modelo particiona tus datos en patrones y residuos**.
-We will take advantage of the fact that you can think about a model partitioning your data into pattern and residuals. Encontraremos patrones con visualizaciones, luego los haremos concretos y precisos con un modelo. Repetiremos luego el proceso, pero reemplazaremos la variable antigua con los residuos del modelo. The goal is to transition from implicit knowledge in the data and your head to explicit knowledge in a quantitative model. This makes it easier to apply to new domains, and easier for others to use. 
+Tomaremos ventaja del hecho que se puede (que podemos) pensar que un modelo particiona tus datos en patrones y residuos. Encontraremos patrones con visualizaciones, luego los haremos concretos y precisos con un modelo. Repetiremos luego el proceso, pero reemplazaremos la variable antigua con los residuos del modelo. The goal is to transition from implicit knowledge in the data and your head to explicit knowledge in a quantitative model. This makes it easier to apply to new domains, and easier for others to use. 
```

Comment:
Traducir *We will take advantage of the fact that you can think about a model* como 
*Tomaremos ventaja del hecho de que **puedes** pensar que un modelo...*